packages/store/src/execution/dispatchOutsideZoneNgxsExecutionStrategy.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  NgZone,
+  PLATFORM_ID,
+  Injectable,
+  ɵNoopNgZone as NoopNgZone,
+  Inject
+} from '@angular/core';
+import { isPlatformServer } from '@angular/common';
+
+import { NgxsExecutionStrategy } from '../symbols';
+import { ConfigValidator } from '../internal/config-validator';
+
+@Injectable()
+export class DispatchOutsideZoneNgxsExecutionStrategy implements NgxsExecutionStrategy {
+  constructor(private _ngZone: NgZone, @Inject(PLATFORM_ID) private _platformId: Object) {
+    this.verifyZoneIsNotNooped(this._ngZone);
+  }
+
+  enter<T>(func: (...args: any[]) => T): T {
+    if (isPlatformServer(this._platformId)) {
+      return this.runInsideAngular(func);
+    } else {
```

Comment:
You can get rid of the else and just leave the return statement.

Reply:
Will do.

Reply:
Done